packages/sdk-rtl/src/baseTransport.ts

```diff
@@ -189,11 +189,11 @@ export abstract class BaseTransport implements ITransport {
           );
           console.debug({ AbortSignal });
         }
-      } else {
-        console.debug(
-          'AbortSignal.timeout is not defined. Timeout will use default behavior'
-        );
       }
+    } else {
+      console.debug(
+        'AbortSignal.timeout is not defined. Timeout will use default behavior'
+      );
```

Comment:
This log was being generated even if AbortSignal.timeout was available in the runtime, since it was attached to the wrong condition (options.signal)